Makefile

```diff
@@ -53,21 +53,19 @@ build-clean: clean
 # build a dist zip file from the distdir, THEN run the tests in the dist dir,
 # to avoid polluting the distfile with dev dependencies
 ###
-dist: dist-zip dist-test
+dist: dist-test
```

Comment:
Why do we remove `dist-zip`?  according to the comments above, we need to build a dist zip.

Reply:
`dist-test` also calls `dist-zip`, so calling both is a bit redundant.